blog/2022-09-15-tauri-1.1.mdx

```diff
@@ -0,0 +1,179 @@
+---
+authors: [lucasfernog]
+image: ./tauri_1_1_images/header.png
+description: After 113 pull requests and nearly two months of work, the Tauri team is pleased to announce the 1.1.0 release.
+---
+
+# Announcing Tauri 1.1.0
+
+![Tauri 1.0 Launch Hero Image](./tauri_1_1_images/header.png)
+
+After 113 pull requests and nearly two months of work, the Tauri team is pleased to announce the 1.1.0 release. The changes were internally audited and no security issues were found.
+
+You can update the NPM dependencies with:
+
+```
+$ npm update
+```
```

Comment:
Do you want me to add a command tab switcher thing to this to include Yarn and pnpm? Is it `yarn upgrade` and `pnpm update`?

Reply:
we could also link to the "update dependencies" guide tho that one needs changes too :joy: